mono/metadata/unity-utils.c

```diff
@@ -1062,3 +1062,15 @@ mono_unity_class_get_generic_parameter_count (MonoClass* klass)
 
 	return klass->generic_container->type_argc;
 }
+
+MONO_API MonoClass*
+mono_unity_class_get(MonoImage* image, guint32 type_token)
+{
+	// Unity expects to try to get classes that don't exist, and
+	// have a value of NULL returned. So eat the error message.
+	MonoError unused;
+	MonoClass* klass= mono_class_get_checked(image, type_token, &unused);
+	mono_error_cleanup(&unused);
```

Comment:
I've fixed this (with a force push of a single new squashed commit). My kingdom for RAII in C.